content/de/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,179 @@
+---
+reviewers:
+- bgrant0607
+- mikedanese
```

Comment:
Can you remove this block (`reviewers` section), it is not needed. When we don't specify this, the `sig-docs-de-reviewers` will be used.